action.yml

```
@@ -24,4 +27,7 @@ runs:
         max_attempts: 3
         command: |
           chmod +x $GITHUB_WORKSPACE/.github/workflows/post_to_mastodon.py
-          $GITHUB_WORKSPACE/.github/workflows/post_to_mastodon.py
+          $GITHUB_WORKSPACE/.github/workflows/post_to_mastodon.py \
+            --message "${{ inputs.message }}" \
+            --access-token ${{ input.access-token }} 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)